Proposal - Adjust the BAT-A Debt Ceiling - Sepember 17, 2020.md

```diff
@@ -0,0 +1,37 @@
+---
+title: Increase the BAT-A Debt Ceiling - September 17, 2020
+summary: Signal your support to increase the BAT Debt Ceiling to 10 million dai
```

Comment:
Should be 'support or opposition' Fix capitalisation on Dai. Also please do the before-after. So 'increase from 5 million Dai to 10 million Dai'